pkg/v1/tkg/test/tkgctl/shared/e2e_common.go

```diff
@@ -13,17 +13,15 @@ import (
 	"strings"
 	"time"
 
-	"sigs.k8s.io/cluster-api/util"
```

Comment:
Note: as ```e2e_common.go``` is used by all pre-utkg tests as well, its better not to add utkg specific tests here, as it cause clashes when certain functionality get called from there, for example when resources functionalities interacting with specific version of YanzuKubernetesRelease be called, etc.

Because of that reverting this file to almost what it was prior to #3106 and define a new file for utkg related tests